### PR TITLE
add new get started

### PR DESCRIPTION
**Pull request by alexgeron-Luos**

### CODE REVIEW

File: get-started/get-started1.md [hunk 1]
```diff
@@ -9,96 +9,138 @@ import IconExternalLink from '@theme/IconExternalLink';
...
-The project folder is opened in the explorer.
+:::caution
+In some cases, the build may output an error message, you may need to update the PlatformIO libraries and rebuild as explained in this image:
 
-:::note Note
-Arduino users can select their board on the `platformio.ini` file by modifying the `board = zero` line.
-:::
+<div align="center">
+  <Image src="/img/get-started/get-started-1-5.png" darkSrc=''/>
+</div>
```
Comment (Simonbdy) [inline get-started/get-started1.md:99]: I see no image on the rendering page

File: get-started/get-started2.md [hunk 2]
```diff
@@ -56,18 +77,62 @@ Your luos device have been successfully mounted into a "device" object:
...
+```
+In [5]:device.blinker.play()
+```
+
+:::info
+You can always check the list of the commands available for all services using your `tab` key:
+
+<div align="center">
+  <Image src="/img/get-started/get-started-2.png" darkSrc=''/>
+</div>
```
Comment (Simonbdy) [inline get-started/get-started2.md:130]: No image displaying

File: get-started/get-started3.md [hunk 3]
```diff
@@ -3,110 +3,202 @@ custom_edit_url: null
...
 
-In this tutorial, we will use the default wiring defined by Luos. The customization of your physical interface will be addressed in another tutorial.
-The default hardware interface used by Luos is defined on the [LuosHAL](https://github.com/Luos-io/LuosHAL) folder corresponding to your device.
+:::
 
-Luos needs a broadcast communication; in this example, we will create a OneWire network limiting the circuit to simple wires.
-Luos also needs a Point To Point (PTP) connection allowing to define the physical position of your boards. In this example, we will have two PTP lines per board.
+<div align="center">
+  <Image src="/img/get-started/get-started-3.png" darkSrc=''/>
+</div>
```
Comment (Simonbdy) [inline get-started/get-started3.md:54]: No image

File: get-started/get-started3.md [hunk 4]
```diff
@@ -3,110 +3,202 @@ custom_edit_url: null
...
+</TabItem>
+<TabItem value="STM32F401RE/STM32F410RB" label="STM32F401RE/STM32F410RB">
 <div align="center">
-  <Image src="/img/Get_started_board_connection_black.png" darkSrc="/img/Get_started_board_connection_white.png"/>
+  <Image src="/img/get-started/get-started-3-5.png" darkSrc=''/>
 </div>
+</TabItem>
+</Tabs>
+</TabItem>
+</Tabs>
```
Comment (Simonbdy) [inline get-started/get-started3.md]: The table doesn't display, nor the images

File: get-started/get-started3.md [hunk 5]
```diff
@@ -3,110 +3,202 @@ custom_edit_url: null
...
+
 :::
 
-### Use this network!
+4. Check if the proper board environnement is selected depending on your board:
 
-From the [first _Get started_ tutorial](/get-started/get-started), you should have a _get_started_ repository on your computer. We will use this code to demonstrate how Luos works using a network.
+  <div align="center">
+  <Image src="/img/get-started/get-started-3-6.png" darkSrc=''/>
+</div>
```
Comment (Simonbdy) [inline get-started/get-started3.md:144]: No image

File: get-started/get-started3.md [hunk 6]
```diff
@@ -3,110 +3,202 @@ custom_edit_url: null
...
+
+:::
 
-Now flash _board 2_, and it's done!
+4. Check if the proper board environnement is selected depending on your board:
 
-To check if everything is OK, plug a USB cable into _board 1_. The _board 1_ LED should blink thanks to the _board 2_ app.
+<div align="center">
+  <Image src="/img/get-started/get-started-3-7.png" darkSrc=''/>
+</div>
```
Comment (Simonbdy) [inline get-started/get-started3.md:183]: No image

File: get-started/get-started.md [hunk 7]
```diff
@@ -3,16 +3,67 @@ hide_table_of_contents: true
...
+[Part 3: Unleash your code](/get-started/get-started3)
+
+## 4. Keywords
+
+- Board (Electronic board with an MCU on it)
+- MCU (Microcontroller)
+- Service
+- Network
+- Distributed system
+
```
Comment (Simonbdy) [inline get-started/get-started.md:66]: Can we add hover-over definitions for each word of this list?

**Review by Salem-Luos — APPROVED** [review]

**Review by Simonbdy — APPROVED** [review]
Ok for me

**Pull request merged**